src/mixer.cpp

```diff
@@ -34,23 +34,204 @@
 #include "rosflight.h"
 
 #include <cstdint>
+// We don't have a method to print the mixer currently being used for the user's benefit.
```

Comment:
Should this be resolved before merging? We could just add it to the TODOs for the project.